html-css/week-03/lesson.md

```diff
@@ -228,6 +231,7 @@ Checkboxes that are related to each other can be grouped together in a `<fieldse
     English
   </label>
 </fieldset>
+</form>
```

Comment:
This looks good, thanks. Can you up the indentation for the nested tags in both examples?

Reply:
yeah sure!